### PR TITLE
Correcting serialization of event ends date

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Argh, same memory problem as #98. Wasn't expecting to see it as the schema hasn't been updated here.

File: project_tests.sh [hunk 1]
```diff
@@ -10,4 +10,4 @@ if [ "$dependencies" = "lowest" ]; then
 else
     composer1.0 update --no-interaction
 fi
-vendor/bin/phpunit --log-junit="build/${dependencies}-phpunit.xml"
+vendor/bin/phpunit -dmemory_limit=2G --log-junit="build/${dependencies}-phpunit.xml"
```
Comment (thewilkybarkid) [inline project_tests.sh:13]: That makes me sad.

Comment (giorgiosironi) [reply project_tests.sh:13]: Looks like it's not enough, let me experiment.

Comment (giorgiosironi) [reply project_tests.sh:13]: It halts on a particular test even with higher memory limits, so I'll investigate.

Comment (thewilkybarkid) [reply project_tests.sh:13]: Schema's changed so this can be reverted now.

**Pull request merged**